atomic_reactor/plugins/exit_koji_import.py

```diff
@@ -633,3 +504,215 @@ def run(self):
                        json.dumps(build_info, sort_keys=True, indent=4))
 
         return build_id
+
+
+class KojiImportPlugin(KojiImportBase):
+
+    key = PLUGIN_KOJI_IMPORT_PLUGIN_KEY  # type: ignore
+
+    def __init__(self, tasker, workflow,
+                 kojihub=None, url=None,
+                 verify_ssl=True, use_auth=True,
+                 koji_ssl_certs=None, koji_proxy_user=None,
+                 koji_principal=None, koji_keytab=None,
+                 blocksize=None,
+                 target=None, poll_interval=5):
+        super(KojiImportPlugin, self).__init__(tasker, workflow,
+                                               kojihub, url,
+                                               verify_ssl, use_auth,
+                                               koji_ssl_certs, koji_proxy_user,
+                                               koji_principal, koji_keytab,
+                                               blocksize,
+                                               target, poll_interval)
+
+    def set_media_types(self, extra, worker_metadatas):
+        media_types = []
+        for platform in worker_metadatas:
+            annotations = get_worker_build_info(self.workflow,
+                                                platform).build.get_annotations()
+            if annotations.get('media-types'):
+                media_types = json.loads(annotations['media-types'])
+                break
+        super(KojiImportPlugin, self).set_media_types(extra, worker_metadatas,
+                                                      media_types)
```

Comment:
Rather than passing the extra media types to the base method, could we instead call the base method first then adjust the content of `extra['image']['media_types']` here as necessary? Or even adjust extra first, then call the base method to finalize it?

Reply:
I realized after I went to bed last night that I should refactor `get_build` to be "inside out" from the way it is now :)

Reply:
@chmeliik, @athos-ribeiro, @MartinBasti; Can one of you PTAL - I've refactored `set_media_types`, and now AFAICT it's not even being called, and I can't figure out why...

The pytests from Travis should show the same failure I'm seeing.